### DISCUSSION TITLE
Youtube Private/Unlisted video embed

### DISCUSSION THREAD

**Discussion by sheiquezakir** [GitbookIO/community#510] [0. General bugs and questions] [answered]
Is it possible to add youtube Private or Unlisted video in Gitbook?

I have a video that I don't want to share with anyone, but it should be visible only on GitBook.

**Comment by addisonschultz** [comment, marked as answer]
Hi @sheiquezakir ! You can embed unlisted videos on GitBook - all you have to do is paste in the link in the GitBook editor!

**Comment by sheiquezakir** [comment]
Thank you! Could you tell how to remove Pop-out option in the video player. I'm using Google video link.

**Reply by addisonschultz** [reply]
Can you give me an example of the link you're pasting? 

The embedded experience is set by the service you're using - so YouTube in this instance! They have a list of available parameters you can check out here: https://developers.google.com/youtube/player_parameters

If they do not have an option to remove the Pop-out player, then unfortunately it's not possible

**Comment by sheiquezakir** [comment]
I used this link

https://drive.google.com/file/d/1Ue0pbXFOb5IeCJZg7hQBGg5EtSiQf6GS/view?usp=drive_link

<img width="889" alt="Screenshot 2024-03-12 at 2 06 10 AM" src="https://github.com/GitbookIO/community/assets/69828835/45e3d4f5-b957-48d6-9e5e-64f82279f897">

Is it possible to upload my video to GitBook itself and play it from there, ensuring its privacy? If you have any alternative solutions for keeping my video private and restricted to playback only within GitBook, that would be greatly appreciated.

**Reply by addisonschultz** [reply]
Unfortunately, it isn't possible to upload video files directly to GitBook - you'll need to host it somewhere first!

I would suggest uploading this video to YouTube as unlisted, and embed it from there 👍

**Reply by sheiquezakir** [reply]
I can see the share option in the YouTube player as well; please refer to the screenshot.

Video link :- https://www.youtube.com/watch?v=N9qiUgA-hlg

![Uploading Screenshot 2024-03-12 at 2.18.11 AM.png…]()

**Reply by addisonschultz** [reply]
I don't think the screenshot uploaded - what do you mean exactly by the share option?

Please refer to the guide here: https://developers.google.com/youtube/player_parameters

If they don't have a parameter to remove it, then unfortunately right now it isn't possible natively in YouTube 😕

**Reply by sheiquezakir** [reply]
Uploading the screenshot again ...
<img width="874" alt="Screenshot 2024-03-12 at 2 25 36 AM" src="https://github.com/GitbookIO/community/assets/69828835/e39667c7-487f-43ea-9d97-74ccde3f4d5e">

**Reply by addisonschultz** [reply]
You will have to refer to https://developers.google.com/youtube/player_parameters 😄

**Reply by sheiquezakir** [reply]
They are not supporting it. Do you guys have any plan/roadmap to make the video private?

https://developers.google.com/youtube/player_parameters#august-23,-2018

**Reply by addisonschultz** [reply]
It should already work, if you are planning on making the video unlisted!

**Reply by sheiquezakir** [reply]
<img width="1255" alt="Screenshot 2024-03-12 at 10 31 01 AM" src="https://github.com/GitbookIO/community/assets/69828835/016ab05b-4076-42b8-bd5f-74895d2b6b53">

This video is already marked as unlisted
